reflex/components/radix/themes/typography/link.py

```diff
@@ -22,12 +25,17 @@
 
 LiteralLinkUnderline = Literal["auto", "hover", "always"]
 
+next_link = NextLink.create()
+
 
 class Link(CommonMarginProps, RadixThemesComponent, A):
     """A semantic element for navigation between pages."""
 
     tag = "Link"
 
+    # What the link renders to.
+    as_: Var[str] = BaseVar.create(value="{NextLink}", _var_is_local=False)  # type: ignore
+
```

Comment:
in radix, we have to set `as_child=True` and pass the `NextLink` wrapping the children

```python
def link(*children, href: str, **props):
    """Create a link."""
    return rdxt.link(rx.next_link(*children, href=href, **props), as_child=True)
```